### PR TITLE
feat: add show more button to request variables and path params

### PR DESCRIPTION
**Pull request by marclave**
fixes https://github.com/scalar/scalar/discussions/731

before 😮‍💨 
image

after 😎 

image

### CODE REVIEW

File: packages/api-reference/src/helpers/getParametersFromOperation.ts [hunk 1]
```diff
@@ -21,25 +21,32 @@ export function getParametersFromOperation(
...
+      value: parameter.example
+        ? parameter.example
+        : parameter.schema
+          ? getExampleFromSchema(parameter.schema, { mode: 'write' })
+          : '',
+      required: parameter.required ?? false,
+      enabled: parameter.required ?? false,
+    }))
+
+  return params.sort((a, b) => {
```
Comment (amritk) [inline packages/api-reference/src/helpers/getParametersFromOperation.ts:44]: excellent comments

**Review by amritk — APPROVED** [review]
Code looks good 👍🏾 

Did not test as I didn't see a preview. Guess I can pull and test like the stone ages... 😉

**Review by hwkr — APPROVED** [review]
Code looks good! From a design / accessibility standpoint I'm a little worried about how little contrast there is on the show more button.

I kind of had to hunt for it in the before and after and I was looking for something different 😅 Also it's tucked in the corner and the only affordance it has to show it's a button is the chevron in the corner.

Google Chrome-2024-02-13-13-56-33@2x

**Comment by marclave** [comment]
> Code looks good! From a design / accessibility standpoint I'm a little worried about how little contrast there is on the show more button.
> 
> I kind of had to hunt for it in the before and after and I was looking for something different 😅 Also it's tucked in the corner and the only affordance it has to show it's a button is the chevron in the corner.
> 
> Google Chrome-2024-02-13-13-56-33@2x
> Maybe the button could have a border to make it feel a little bit more like a button? My 2 cents as accessibility gremlin 👹
> 
> Google Chrome-2024-02-13-13-59-49@2x

great feedback! thanks for checking the contrast. this could be part of a larger discussion, we need to get this fix in but you me and @cameronrohani should plan out stronger accessibility guidelines to align moving forward : ) cause it's really important ✨

**Pull request merged**